internal/service/encryptionatrestprivateendpoint/resource_test.go

```diff
@@ -70,6 +71,53 @@ func basicTestCase(tb testing.TB) *resource.TestCase {
 	}
 }
 
+func TestAccEncryptionAtRestPrivateEndpoint_approveEndpointWithAzureProvider(t *testing.T) {
+	acc.SkipTestForCI(t) // uses azure/azapi Terraform provider which can log sensitive information in CI like Azure subscriptionID used in parent_id of the resource
```

Comment:
if we know the patterns, GH allows to hide sensitive info, e.g.: https://github.com/marketplace/actions/hide-sensitive-inputs

but maybe it's not suitable in this situation

Reply:
The issue is we don't need to hide once specific attribute here that is a direct input to the action. The attribute that's concerning is the parent_id which is constructed/initialized inside the TF config. 
Then when a failure happens, the azure provider includes that value in it's failure message. So essentially it's the failure message that can log that information.

Does that make sense?